gauntlet/packages/gauntlet-solana-contracts/artifacts/schemas/ocr2.json

```diff
@@ -845,10 +830,7 @@
           {
             "name": "xs",
             "type": {
-              "array": [
-                "u8",
-                4096
-              ]
+              "array": ["u8", 4096]
```

Comment:
This field is called `Raw` not `xs` in `pkg/solana/types.go#OffchainConfig` - and therefore in the program's state as well. Is it worth renaming it here for consistency?

Reply:
The `IDL` schemas are autogenerated with `anchor build`